addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1729,10 +1722,14 @@ options.registry.Carousel = options.Class.extend({
     /**
      * @override
      */
-    notify: function (name, data) {
+    async notify(name, data) {
```

Comment:
notify should not be async. If there are async parts inside, make sure they are in the mutex. If this is only to use `await`, don't :wink:

Reply:
Ah I didn't know :grimacing: I will fix that

> If this is only to use await, don't 

Do you mean to just remove the "await" word ?

Reply:
> Do you mean to just remove the await word ?

Probably replacing it by a `.then`